### PR TITLE
refactor: stop using ActiveSupport

### PR DESCRIPTION
**Pull request by olleolleolle**
Closes: n/a

# Goal

In order to have an even smaller dependency surface, let's remove the ActiveSupport library.

# Approach
1. Remove the gem dependency.
2. Replace `ActiveSupport::JSON.decode` with `JSON.parse`.
3. Add `pry` to facilitate debugging.

### CODE REVIEW

**Comment by ivanoblomov** [comment]
What happened to this one?

**Comment by olleolleolle** [comment]
I made an attempt at getting a key using the description in the readme and the landscape had changed - there is now about 20 kinds of key to ask for most of them are in enterprise products in Maps. I had no idea how to get a usable key of my own.

**Comment by ivanoblomov** [comment]
Gonna take a stab at this.

**Comment by ivanoblomov** [comment]
@olleolleolle take a look.

File: lib/google_maps_geocoder/google_maps_geocoder.rb [hunk 1]
```diff
@@ -183,11 +183,11 @@ def parse_formatted_street_address
   end
 
   def parse_lat
-    @json['results'][0]['geometry']['location']['lat']
+    @json[:results][0][:geometry][:location][:lat]
```
Comment (olleolleolle) [inline lib/google_maps_geocoder/google_maps_geocoder.rb]: Minor: Another refactoring would be to use Hash#dig and Array#dig for these long ones.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'pry'
```
Comment (olleolleolle) [inline spec/spec_helper.rb:3]: Minor: Perhaps we are keeping Pry, it's nice to have access to, when fiddling with things?

**Review by olleolleolle — COMMENTED** [review]
Thanks for fixing it!

**Pull request merged**